infra/main.bicep

```diff
@@ -436,6 +450,33 @@ module storage 'core/storage/storage-account.bicep' = {
   }
 }
 
+
+module userStorage 'core/storage/storage-account.bicep' = if (useUserUpload) {
+  name: 'user-storage'
+  scope: storageResourceGroup
+  params: {
+    name: !empty(userStorageAccountName) ? userStorageAccountName : 'user${abbrs.storageStorageAccounts}${resourceToken}'
+    location: storageResourceGroupLocation
+    tags: tags
+    allowBlobPublicAccess: false
+    publicNetworkAccess: 'Enabled'
+    isHnsEnabled: true
+    sku: {
+      name: storageSkuName
+    }
+    deleteRetentionPolicy: {
```

Comment:
is this an arbitrary choice? Or should we just remove the delete retention policy for now

Reply:
Arbitrary, can delete.